### PR TITLE
Fixes #947 Input is not exported in TypeScript definitions

### LINKED ISSUE #947: Input is not exported in TypeScript definitions
### Description

**1. Explain what you did**

I've updated from 0.19 to 1.0.0-beta to change the style of my Searchbar (works). Had to replace FormInput with Input to avoid the Invariant-Violation-Error.

**2. What you expected to happen**

I can still import FormInput, FormLabel etc. from the beta-version (index.d.ts), but 'Input' can't be found. 

`import {Text, Button, Input, Icon, Card} from 'react-native-elements';`

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -730,11 +730,70 @@ export interface FormValidationMessageProps extends ViewProperties {
...
+export interface InputProps {
+  /**
+     * Styling for the input container
+     */
+  containerStyle?: StyleProp<ViewStyle>;
+
+  /**
+    * Left icon
+    */
+  leftIcon?: JSX.Element;
```
Comment (msevestre) [inline src/index.d.ts]: I am not sure about that this property. Let me know if this should be changed

**Comment by msevestre** [comment]
@iRoachie Ok thanks for the info. I also adjusted the type descriptions to match the doc descriptions

**Comment by msevestre** [comment]
@iRoachie Sorry man. Hopefully good to go now

**Comment by iRoachie** [comment]
Thanks again @msevestre

**Pull request merged**